rpg.py

```diff
@@ -1,12 +1,35 @@
+import render
+import util
+
 class Player:
     max_hp = 100
-    def __init__(self, name, gender, max_hp, health, fatigue, defence, strength, speed, intelligence):
+    def __init__(self, name, gender, level, xp, max_hp, health, fatigue, defence, strength, speed, intelligence):
         self.name = name
         self.gender = gender
+        self.level = level
+        self.xp = xp
         self.max_hp = max_hp
         self.health = health
         self.fatigue = fatigue
         self.defence = defence
         self.strength = strength
         self.speed = speed
-        self.intelligence = intelligence
\ No newline at end of file
+        self.intelligence = intelligence
+
+    def display_player_bar(self, level, xp, max_hp, health, fatigue):
+        # print xp/level
+        print("LV. " + str(util.calculate_current_level(self.xp)) + " (XP: " + str(self.xp) + "/" + str(round(util.xp_required_for_level(self.level))) + ")", end="")
```

Comment:
Pretty sure this wont work because you're trying to use the current level. Try
```suggestion
        current_level = util.calculate_current_level(self.xp)
        required_xp = util.xp_required_for_level(current_level + 1)
        print("LV. " + str(current_level) + " (XP: " + str(self.xp) + "/" + str(required_xp) + ")", end="")
```

Reply:
I've implemented the change, thanks